content/zh/docs/kitex/Tutorials/advanced-feature/metainfo.md

```diff
@@ -88,6 +88,14 @@ func main() {
         val, ok := metainfo.RecvBackwardValue(ctx, "something-from-server") // 获取服务端传回的元数据
         println(val, ok)
     }
+
+    // 获取服务端传回的所有元数据
+	m := metainfo.RecvAllBackwardValues(ctx) // m: map[string]string
+    if m != nil {
+		for key, value := range m {
+			log.Printf("key: %s, value: %s", key, value)
+		}
+	}
```

Comment:
mixing tabs and spaces: (same above)
![image](https://github.com/cloudwego/cloudwego.github.io/assets/367085/ee52701e-4d63-4986-8be8-45dc0268cf9c)

Reply:
fixed.